src/options/options.css

```diff
@@ -20,74 +19,69 @@ body.mobile .mobile-incompatible {
   opacity: 0.4;
 }
 
-ul {
-  margin: 0px;
-  padding: 0px;
-}
-li {
-  list-style-type: none;
-  margin-top: 10px;
-  padding: 0px;
+form {
+  margin: 16px 32px;
 }
-/* in a fieldset e.g. we use a condensed version */
-.condensed-list {
-  margin-left: 0px;
+
+section {
+  margin-bottom: 32px;
 }
 
-fieldset {
-  /* same border style as separators above */
-  border: 1px solid;
-  border-radius: 4px;
-  border-color: var(--grey-30); /* Firefox: --in-content-box-border-color */
+h1 {
+  margin: 32px 0 16px 0;
+  padding: 16px 0 0 0;
 
-  /* a big width, but not too big */
-  display: inline-block;
-  width: auto;
-  min-width: 60vw;
+  /* title30 */
+  font-size: 22px;
+  font-weight: 300;
 
-  margin-bottom: 4px;
+  border-top: 1px solid rgba(12, 12, 13, 0.15);
 }
 
-/* when a setting has a label *before* it, add some margin */
-label + .setting {
-  margin-left: 8px;
+h1.first {
```

Comment:
Hmm, that markup in the HTML is not so nice…

Can't we just use some pseudo-selectors of CSS in a clever way to achieve this?
Kinda like: https://stackoverflow.com/questions/2717480/css-selector-for-first-element-with-class

I'll guess find a good way… :wink: